src/nc_message.c

```diff
@@ -329,7 +329,7 @@ msg_get_error(bool redis, err_t err)
     struct mbuf *mbuf;
     int n;
     char *errstr = err ? strerror(err) : "unknown";
-    char *protstr = redis ? "-ERR" : "SERVER_ERROR";
+    char *protstr = redis ? "-ERR" : "ERROR";
```

Comment:
Changing this string doesn't account for the source or type of the error, does it? In the memcached protocol, there are three types of errors, ERROR, CLIENT_ERROR, and SERVER_ERROR. This makes the assumption that all errors are server errors and doesn't actually handle the case, per the memcached protocol, that severe errors result in connection termination (which is what twemproxy does by default)